src/demux/handlers/mysql/testHelpers/docker.ts

```diff
@@ -0,0 +1,84 @@
+import { Container } from "dockerode"
+import { Stream } from "stream"
+
+export function promisifyStream(stream: Stream): Promise<string> {
+  const data: Buffer[] = []
+  return new Promise((resolve, reject) => {
+    stream.on("data", (chunk) => data.push(chunk))
+    stream.on("end", () => {
+      const toReturn = Buffer.concat(data).toString()
+      resolve(toReturn)
+    })
+    stream.on("error", () => reject())
+  })
+}
+
+export async function pullImage(docker: any, imageName: string) {
+  const stream = await docker.pull(imageName)
+  await promisifyStream(stream)
+}
+
+export async function waitForMysql(_container: Container, _dbPass: String) {
+  await wait(20000);
+  //let connectionTries = 0
```

Comment:
if this is commented out we should remove it.

Reply:
agreed, but some better method than waiting 20 seconds is needed as well. I'll look into a more deterministic command